app/controllers/organization_invites_controller.rb

```diff
@@ -0,0 +1,27 @@
+class OrganizationInvitesController < ApplicationController
+  def new
+    @organization = Organization.find(params[:organization_id])
+  end
+
+  def create
```

Comment:
I know we already talked about this, but this one will be a great one to refactor.